java/arcs/sdk/android/storage/ServiceStore.kt

```diff
@@ -176,6 +189,7 @@ class ServiceStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     fun onLifecycleDestroyed() {
         serviceConnection?.disconnect()
         storageService = null
-        scope.coroutineContext[Job.Key]?.cancelChildren()
+        channel = null
```

Comment:
Do we need an additional channel?.close() or channel?.cancel() here before assigning null?